electron-builder.yml

```diff
@@ -1,17 +1,24 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-appId: com.aimobile.app
+appId: com.aiandroid.app
 directories:
     app: drop/electron/electron
+    buildResources: src/electron/resources
```

Comment:
Curious if this cannot read the License file directly? the one that is already in our repo

Reply:
@Shobhit1 text is different - the one in our repo refers to AI for Web

Reply:
yeah, plus as mentioned here https://www.electron.build/configuration/dmg
you have to add the license in the buildResources folder and it should be picked up, and to do that we have to define the buildResources folder path